DATA/SHIPS/loadouts.ini

```diff
@@ -3553,7 +3553,7 @@ cargo = ge_s_repair_01, 30
 
 [Loadout]
 nickname = MSN13_Juni
-archetype = li_elite2
+archetype = or_elite
```

Comment:
For the `MSN12_Juni` and `MSN13_Juni` changes, I believe you also have to update the respective `ship_archetype` entries in `MISSIONS/M12/npcships.ini` and `MISSIONS/M13/npcships.ini`.